download_plugins.py

```diff
@@ -1,85 +1,160 @@
-import requests, yaml, sys
+import requests
+import yaml
+import sys
+from textwrap import dedent
+from distutils.version import LooseVersion
+
 
 # Get dependencies for a plugin, or get depedencies for a dependency.
 def get_dependencies(plugin):
-    print "Download dependencies for:", plugin
-    print "*****************************************"
-    print plugin, "has these dependencies:"
+    print(dedent("""
+    Finding dependencies for: {0}
+    ******************************************
+    {0} has these dependencies:""".format(plugin)))
 
```

Comment:
Could you add a flag (e.g `--debug`) for these messages? I think the only important messages should be which plugins are downloads, and the warning messages in the end for now. The rest should not be printed every time.